awscli/customizations/s3/subcommands.py

```diff
@@ -750,12 +750,35 @@ class SyncCommand(S3TransferCommand):
                 [METADATA, METADATA_DIRECTIVE]
 
 
-class MbCommand(S3TransferCommand):
+class MbCommand(S3Command):
     NAME = 'mb'
     DESCRIPTION = "Creates an S3 bucket."
     USAGE = "<S3Uri>"
-    ARG_TABLE = [{'name': 'paths', 'nargs': 1, 'positional_arg': True,
-                  'synopsis': USAGE}]
+    ARG_TABLE = [{'name': 'path', 'positional_arg': True, 'synopsis': USAGE}]
+
+    def _run_main(self, parsed_args, parsed_globals):
+        super(MbCommand, self)._run_main(parsed_args, parsed_globals)
+
+        path = parsed_args.path
+        if path.startswith('s3://'):
```

Comment:
This isn't quite the exact same as the old behavior.  We require the user to provide the `s3://` prefix.

While we can discuss whether or not we should start allowing users to drop the `s3://` prefix where it's obviously implied, I don't think that should be the purpose of this PR.  The point here is simply to port this into it's own separate command *without* changing existing behavior.